xml/ay_custom_user_scripts.xml

```diff
@@ -185,11 +185,21 @@
      <emphasis>init-scripts</emphasis>.
     </para>
     <para>
-     The following elements must be between the
-     &lt;scripts&gt;&lt;init-scripts
-     config:type="list"&gt;&lt;script&gt; ...
-     &lt;/script&gt;&lt;/init-scripts&gt;...&lt;/scripts&gt;
-     tags
+     Init scripts elements must be placed as follows:
+    </para>
+<screen>&lt;scripts&gt;
+    &lt;init-scripts config:type="list"&gt;
+      &lt;script&gt;
+        ...
+      &lt;/script&gt;
+    &lt;/init-scripts&gt;
+&lt;/scripts&gt;</screen>
+
+    <para>
+     Init scripts are different from the rest of script types because they are
+     not executed by &yast; but after &yast; has finished. For this reason,
```

Comment:
Hello @imobachgs! It is nice to "see" you again :+1: 
I have a few small corrections, otherwise this looks good.
`not executed by &yast;` needs a comma after &yast;`

Reply:
Hi @cjschroder! My pleasure :)